src/Orleans/CodeGeneration/GrainInterfaceData.cs

```diff
@@ -424,7 +520,7 @@ private static bool ValidateInterfaceProperties(Type type, List<string> violatio
         {
             bool success = true;
 
-            PropertyInfo[] properties = type.GetProperties();
```

Comment:
@sergeybykov Just  to confirm. ValidateInterfaceProperties() currently retrieves all properties and makes sure there are no properties defined and gives compile error if there are any. Should we remove that check?